capitalize_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'base_decorator'
+
+require_relative 'person'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super().capitalize!
+  end
```

Comment:
- [ ] When you use `super()` in this line you will get the below error:
![afbeelding](https://user-images.githubusercontent.com/39240820/234520100-1463d367-6bc1-4077-a945-a876e88957c0.png)
to fix this please use the instance variable `@nameable` and call the correct_name method like below, and also make sure to first check if the name is longer than 10 or not.
```
 def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end

```